tox.ini

```diff
@@ -62,6 +60,8 @@ deps =
     mypy-zope
     ; Libraries which include type annotations:
     hypothesis
+    pytest>=6.0.0
```

Comment:
```suggestion
    pytest>=6.1.1
```

Reply:
Why 6.1.1? If I read the ChangeLog correctly, the 6.0.0 release should also have the type annotations.

Reply:
@mthuurne unlike production deps, there's no benefit in supporting a wide range of test deps. Might as well upgrade to the latest

Reply:
6.0.0 represents the feature we depend upon, while 6.1.1 represents the time at which this dependency was added. The former feels more important to me. In practice, pip installs the latest version when you set up a virtual env, so I'm not sure the difference really matters.